multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -56,6 +76,29 @@ coreos:
         [Install]
         WantedBy=multi-user.target
 
+{{ if eq .ContainerRuntime "rkt" }}
+    - name: rkt-api.service
+      command: start
+      enable: true
+      content: |
+        [Service]
+        ExecStart=/usr/bin/rkt api-service
+        Restart=always
+        RestartSec=10
+
+    - name: load-rkt-stage1.service
+      command: start
+      content: |
+        [Unit]
+        Description=Load rkt stage1 images
+        Documentation=http://github.com/coreos/rkt
+        Requires=network-online.target
+        After=network-online.target
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/rkt fetch /usr/lib/rkt/stage1-images/stage1-coreos.aci /usr/lib/rkt/stage1-images/stage1-fly.aci  --insecure-options=image
```

Comment:
It appears I mis-understood. It will be cached after first use and we don't need this.

Telling the kubelet to use a stage1 from a url over a filepath was enough to fix the caching. This service was never necessary, I just copied it from our upstream configs.